conf/ynh-vpnclient

```diff
@@ -200,6 +200,20 @@ stop_openvpn() {
 sync_time() {
   systemctl stop ntp
   timeout 20 ntpd -qg &> /dev/null
+  
+  # Some networks drop ntp port (udp 123). 
+  # Try to get the date with an http request on the internetcube web site
+  if [ $? -ne 0 ]; then
+    http_date=`curl -sD - labriqueinter.net | grep '^Date:' | cut -d' ' -f3-6`
+    http_date=`date -d "${http_date}" +%s`
```

Comment:
The result of this command should be stored in a separate variable, `http_date_seconds` for example.